Apollo/DeviceViewers/LoopViewer.xaml

```diff
@@ -0,0 +1,21 @@
+
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             x:Class="Apollo.DeviceViewers.LoopViewer"
+             xmlns:Components="clr-namespace:Apollo.Components"
+             Margin="10 5"
+             DetachedFromVisualTree="Unloaded">
+  <Grid RowDefinitions="*,*" Width="125">
+    <Border Grid.Row="0" BorderBrush="{DynamicResource ThemeBorderHighBrush}" BorderThickness="0 0 0 1">
+      <Grid ColumnDefinitions="*,*" Margin="5 5 5 0">
```

Comment:
Swap the rows maybe? Usually Duration/Gate ends up on the bottom of the device, see Copy/Fade...? (might be nitpicky, but for consistency sake)